meson.build

```diff
@@ -0,0 +1,85 @@
+project('dbrew', 'c', 'cpp', version: '1.0.0', license: 'LGPL-v2',
+        default_options: ['c_std=gnu11', 'cpp_std=gnu++14'])
+
+# Add warning flags
+main_lib_args = ['-Wall', '-Wextra', '-Wmissing-field-initializers',
+                 '-Wunused-parameter', '-Wold-style-definition',
+                 '-Wmissing-declarations', '-Wmissing-prototypes',
+                 '-Wredundant-decls', '-Wmissing-noreturn', '-Wshadow' ,
+                 '-Wpointer-arith', '-Wwrite-strings', '-Winline', '-Wformat-nonliteral',
+                 '-Wformat-security', '-Wswitch-default', '-Winit-self', '-Wnested-externs',
+                 '-Wstrict-prototypes', '-Wmissing-include-dirs', '-Wundef',
+                 '-Wmissing-format-attribute']
+
+
+
+cc = meson.get_compiler('c')
+
+add_global_arguments('-fno-pie', language: 'c')
+add_global_link_arguments('-no-pie', language: 'c')
+add_global_arguments('-fno-pie', language: 'cpp')
+add_global_link_arguments('-no-pie', language: 'cpp')
+
+inc = include_directories('include/', 'include/priv', '.')
+
+pkgconf_libs = []
+
+# config.h generation
+conf = configuration_data()
+
+# FIXME: Get rid of this mess. Some progress here: https://github.com/mesonbuild/meson/issues/1611
+llvm_libs = ''
```

Comment:
In my opinion, this belongs into the LLVM sub-directory. DBrew itself does/should not require LLVM.